### PR TITLE
[Tool-32] - Create Order Model and Save Order

### PR DESCRIPTION
**Pull request by mv-mariamkhan**
The following code includes the creation of the Order.cs and OrderDBConfiguration.cs entites for creating the Orders table in the SQL Server database.

Additionally, updates have been made to the OrderService to preserve changes from the New Order form into the database.

JIRA Ticket URL: https://mediavalet.atlassian.net/browse/TOOL-32

### CODE REVIEW

File: src/MediaValetSubMgmt.Infrastructure/Services/Implementation/OrderService.cs [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+                DataCenterCode = orderViewModel.DataCenter,
+                LogoUrl = orderViewModel.LogoUrl,
+                PortalAccentColor = orderViewModel.PortalAccentColor,
+                IntegrationDomainUsers = orderViewModel.IntegrationDomainUsers,
+                DeliveryDate = orderViewModel.DeliveryDate,
+                Comments = orderViewModel.Comments
+            };
+
+            await _orderRepository.AddAsync(order, cancellationToken);
+            await _orderRepository.UpdateAsync(order, cancellationToken);
```
Comment (leocorreamv) [inline src/MediaValetSubMgmt.Infrastructure/Services/Implementation/OrderService.cs:45]: this is not required, only need to call AddAsync

File: src/MediaValetSubMgmt.Data/DataContext/MVDataContext.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public MVDataContext(DbContextOptions options)
         {
         }
 
-        //public DbSet<Order> Orders { get; set; }
+        public DbSet<Order> Orders { get; set; }
 
         public DbSet<OrderType> OrderTypes { get; set; }
```
Comment (leocorreamv) [inline src/MediaValetSubMgmt.Data/DataContext/MVDataContext.cs:24]: missing the DbConfiguration

**Review by leocorreamv — CHANGES_REQUESTED** [review]

File: src/MediaValetSubMgmt.Data/DataContext/MVDataContext.cs (hunk 2 above)
Comment (mv-mariamkhan) [reply src/MediaValetSubMgmt.Data/DataContext/MVDataContext.cs:24]: resolved in latest commit

File: src/MediaValetSubMgmt.Infrastructure/Services/Implementation/OrderService.cs (hunk 1 above)
Comment (mv-mariamkhan) [reply src/MediaValetSubMgmt.Infrastructure/Services/Implementation/OrderService.cs]: Resolved in latest commit

**Review by leocorreamv — APPROVED** [review]

**Pull request merged**